core/src/test/scala/integration/kafka/server/KRaftClusterTest.scala

```diff
@@ -1155,6 +1199,8 @@ class KRaftClusterTest {
   def testAuthorizerFailureFoundInControllerStartup(): Unit = {
     val cluster = new KafkaClusterTestKit.Builder(
       new TestKitNodes.Builder().
+        setSecurityProtocol(SecurityProtocol.PLAINTEXT).
```

Comment:
Could we add default value to `securityProtocol` and `listener` to avoid those duplicate code?

Reply:
But this is what I addressed in https://github.com/apache/kafka/pull/16824#discussion_r1726140300... Or did you mean something else ?

Reply:
Okay, that's a bit awkward. These tests don't use the annotation, so they can't reuse the default values of ClusterTest...

A possible workaround is to add default values for securityProtocol and listener. Additionally, we should add comments to remind developers that the default values need to stay consistent.

@brandboat WDYT?